operator/src/main/java/org/bf2/operator/managers/OperandOverrideManager.java

```diff
@@ -30,6 +32,8 @@ public class OperandOverrideManager {
     public static class OperandOverride {
         public String image;
 
+        public List<EnvVar> env;
```

Comment:
I'm related to the thought about a more declarative api I'm also wondering if the overrides would be better as `Map<String, String>` and only converting to `List<EnvVar>` when rendering final result?

Reply:
At the moment the code supports the possibility of overriding with an environment variable whose source is an EnvVarSource.  That might be a useful option to retain.

Reply:
Fine by me.